packages/mui-material-next/src/InputBase/InputBase.test.js

```diff
@@ -654,23 +628,6 @@ describe('<InputBase />', () => {
 
       expect(getByTestId('adornment')).not.to.equal(null);
     });
-
-    it('should allow a Select as an adornment', () => {
-      render(
-        <TextField
```

Comment:
Was this removed because it's a `TextField` test?

Reply:
Yep - but actually it could be tested with `InputBase`... I've added it back (wonder why it's a `TextField` in the first place)